client/blocks/reader-combined-card/index.jsx

```diff
@@ -165,7 +165,7 @@ class ReaderCombinedCardComponent extends Component {
 					/>
 				</div>
 				{ feedId && <QueryReaderFeed feedId={ +feedId } includeMeta={ false } /> }
```

Comment:
There seem to be a few more unnecessary `includeMeta` in the codebase. Should we remove them too?

Reply:
Removed them in #60660